connector_importer/readme/CREDITS.md

```diff
@@ -0,0 +1 @@
+The migration of this module from 16.0 to 18.0 was financially supported by Camptocamp.
```

Comment:
From the changes in __manifest__.py, i saw the version changed from 17.0 to 18.0

Reply:
I started from v16 and I only guaranteed v18. So I wrote v16 --> v18. (v17 I just put it up to get their automatic migration script =)))